packages/react-magma-dom/src/components/Stepper/ResponsibleStepperContainer.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import {
+  Breakpoint,
+  BreakpointsContainer,
+  BreakpointScreenSize,
+} from '../BreakpointsContainer';
+import {
+  Flex,
+  FlexAlignItems,
+  FlexBehavior,
+  FlexDirection,
+  FlexJustify,
+  FlexWrap,
+} from '../Flex';
+import { Stepper, StepperOrientation, StepperProps } from './Stepper';
+
+interface ResponsibleStepperContainerProps {
+  stepperContent: React.ReactNode | React.ReactNode[];
+  children: React.ReactNode | React.ReactNode[];
+  props: StepperProps;
+}
+
+export const ResponsibleStepperContainer: React.FC<
```

Comment:
We don't really use functional components in Magma. Can we swap this to a class component?